dirac.cfg

```diff
@@ -852,6 +852,162 @@ Resources
     # This option is used in the Job Wrapper and, if set, requires the RequestManagementSystem to be installed
     Tier1-Failover = CERN-FAILOVER,CNAF-FAILOVER
   }
+  # Definition of the sites
+  # See http://dirac.readthedocs.io/en/latest/AdministratorGuide/Resources/site.html
+  Sites
+  {
+    LCG
+    {
+      ##BEGIN SiteConfiguration
+      LCG.CERN.ch
+      {
+        # Local Storages
+        SE = CERN-RAW, CERN-RDST, CERN-USER # (Optional) SEs Local to the site
+
+        # Overwrite definities of StorageElement (discouraged)
+        # or StorageElementGroups for that Site
+        AssociatedSEs
+        {
+          # Tier1-Failover is now only CERN-FAILOVER when running a Job at CERN
+          Tier1-Failover = CERN-FAILOVER
+        }
+        CE = ce503.cern.ch, ce504.cern.ch # List of CEs for th esite (updated by BDII2CSAgent)
```

Comment:
```suggestion
```
This list is no longer needed, cf. the CERN site entry in lbcertifdirac70 (don't look at the others, people just added them unnecessarily).
But I can't find the PR that changed this...

Reply:
This was for RSS, and the PR that removed the need for this entry was made long time ago.